packages/front-end/components/Experiment/VariationsTable.tsx

```diff
@@ -117,6 +117,11 @@ const VariationsTable: FC<Props> = ({
   const hasVisualEditorFeature = hasCommercialFeature("visual-editor");
 
   const visualChangesets = _visualChangesets || [];
+  const hasAnyPositionMutations = visualChangesets.some((vc) =>
+    vc.visualChanges.some(
+      (v) => v.domMutations.filter((m) => m.attribute === "position").length > 0
```

Comment:
Is it possible that `vc.visualChanges` or `v.domMutatioms` are ever undefined? Or are they always initialized to empty arrays? Just wanted to flag this as a potential source of errors, ignore if it's not an issue.